get.sh

```diff
@@ -599,6 +599,12 @@ testJavaVersion()
 		echo "=JAVA VERSION OUTPUT BEGIN="
 		${_java} -version
 		echo "=JAVA VERSION OUTPUT END="
+		if [ "$os" = "windows" ]; then
+			TEST_JDK_HOME=$(cygpath -w "${TEST_JDK_HOME}")
+			echo -e "TEST_JDK_HOME=${TEST_JDK_HOME}" | sed 's/\\/\\\\/g' > ${TESTDIR}/job.properties
```

Comment:
I don't think you want `-e` here. Suppose `${TEST_JDK_HOME}` is `C:\test\foo\bar`, `-e` will map `\t` to a tab which isn't likely to work.